kythe/extractors/bazel/extract.sh

```diff
@@ -65,7 +65,14 @@ fi
 # It is ok if targets fail to build. We build using --keep_going and don't
 # care if some targets fail, but bazel will return a failure code if any
 # targets fail.
-/kythe/bazelisk --bazelrc=/kythe/bazelrc "$@" || true
+if [ -n "$BAZEL_TARGET" ]; then
+  /kythe/bazelisk --bazelrc=/kythe/bazelrc "$@" "$BAZEL_TARGET" || true
```

Comment:
Do we need to use `BAZEL_TARGET` in this branch of the `if`?

Reply:
Yeah, I'm changing the args passed to the script in our yaml generator to not include the targets so we have to pass them explicitly.